core/trino-main/src/main/java/io/trino/memory/MemoryResource.java

```diff
@@ -41,41 +33,21 @@
 public class MemoryResource
 {
     private final LocalMemoryManager memoryManager;
-    private final TaskManager taskManager;
 
     @Inject
-    public MemoryResource(LocalMemoryManager memoryManager, TaskManager taskManager)
+    public MemoryResource(LocalMemoryManager memoryManager)
     {
         this.memoryManager = requireNonNull(memoryManager, "memoryManager is null");
-        this.taskManager = requireNonNull(taskManager, "taskManager is null");
     }
 
     @ResourceSecurity(INTERNAL_ONLY)
-    @POST
+    @GET
     @Produces(MediaType.APPLICATION_JSON)
-    @Consumes(MediaType.APPLICATION_JSON)
-    public MemoryInfo getMemoryInfo(MemoryPoolAssignmentsRequest request)
+    public MemoryInfo getMemoryInfo()
     {
-        taskManager.updateMemoryPoolAssignments(request);
         return memoryManager.getInfo();
     }
 
-    @ResourceSecurity(MANAGEMENT_READ)
-    @GET
-    @Path("{poolId}")
-    public Response getMemoryInfo(@PathParam("poolId") String poolId)
```

Comment:
@dain @findepi 
I am a bit concerned about the removal of this endpoint. What was the target audience? Can it backfire?
`getMemoryInfo` changed from `POST` to `GET` above has the relevant information but it is tagged with `INTERNAL_ONLY` security so cannot be used as a replacement.
And I am also not sure if I can safely change `INTERNAL_ONLY` to `MANAGEMENT_READ` above.

Would appreciate some help here.

Reply:
Maybe I can keep it with paramter and require that caller always has to pass "general" ?

Reply:
I'm not sure who uses that.  Maybe @electrum does?  If you do what to keep it, change the annotation to `@Path("general")`.

Reply:
I think this was for internal monitoring at Facebook. I'm fine with removing it.

Reply:
Thanks guys:)